misc_settings/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+Misc settings to be an anchor for your custom settings
+
+Use case : you want your users find in one a unique dedicated page the settings they need rule their activity over the time.
```

Comment:
LGTM but clarity suggestion

Use case: allow your users to manage any settings that you want to add